Elevel/Elevel.Api/Controllers/AuditionController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using Elevel.Application.Features.AudioFeatures;
+using MediatR;
+
+namespace Elevel.Api.Controllers
+{
+    [Route("[controller]/[action]")]
+    public class AuditionController : BaseApiController
+    {
+        private readonly IMediator _mediator;
+        public AuditionController(IMediator mediator)
+        {
+            _mediator = mediator;
```

Comment:
you already have Mediator for you in BaseApiController class, please use it
+ Add Authorize attribute to you controller